app/pb_kits/playbook/pb_checkbox/_checkbox.jsx

```diff
@@ -13,29 +14,25 @@ type CheckboxProps = {
 
 const Checkbox = ({
   dark=false,
-  tag='label',
-  label,
```

Comment:
The props on the rails kit are:

:dark 
:checked
:icon
:text
:value
:name

Shouldn't they match in the react kit?

Reply:
The ones in here look to be:

:dark   
:name   
:value   
:checked
:label   

So its missing `icon` and `text` but contains a different prop called `label`.